presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -86,6 +87,11 @@ public HiveSessionProperties(HiveClientConfig config)
                         config.getOrcMaxReadBlockSize(),
                         false),
                 booleanSessionProperty(
+                        ORC_LAZY_READ_SMALL_RANGES,
+                        "Experimental: ORC: Read small file segments lazily",
```

Comment:
This is experimental but defaults to true.

Reply:
The plan it to remove the config option if it works... maybe this should be hidden?